### PR TITLE
Chinese translation of the documentation 

### PR DESCRIPTION
**Pull request by leejingbo**
I finished the Chinese translation of the documentaion as I said in issue #25 .

The filename ends with `_zh.md`  .

### CODE REVIEW

**Comment by Leonardo2718** [comment]
LGTM as far as I can tell.

@xliang6 ?

**Comment by xliang6** [comment]
I will review this today.

File: README_zh.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+$ luav luac.out  # 执行Lua字节码 (由 `luac` 生成)
+```
+
+## 许可
+
+Lua Vermelh有三个组件，各自使用各自的许可： 
+
+* 微做修改的PUC-Rio Lua VM 仍使用其原始的许可（MIT许可）
+* Lua Vermelha JIT编译器用下列两种许可:
+  * [Eclipse Public License](http://www.eclipse.org/legal/epl-v10.html)
```
Comment (xliang6) [inline README_zh.md]: FYI, Eclipse OMR is now under the **Eclipse Public License 2.0** and the Apache 2.0 license. The license version recently changed, not a problem with this pull request but a general comment for this lua-vermelha project.

File: doc/zh/building.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+## JIT 编译器
+
+JIT编译器可以灵活地构建成两种不同形式的二进制文件。
+
+默认和推荐的JIT构建方式是构建成静态库(static library)。它就既能链接到VM库又能直接链接到应用程序执行文件里面。
+
+当前支持的另一种方式是把JIT作为目标文件（object file）来链接。这种情况下，内部目标文件（intermediate object file，指每个c/c++文件生成的.o文件）被链接成一个单一的大的目标文件。这个单一目标文件可以被其他目标文件再次链接使用。
+
+当前JIT必须直接链接到VM，所以构建成静态库当前还不被支持。
```
Comment (xliang6) [inline doc/zh/building.md]: 当前JIT必须直接链接到VM (不管使用什么二进制)，所以构建成共享库当前还不被支持。

File: doc/zh/building.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+在JIT Makefile中提供了下列伪目标（phony target），以用来构建不同的二进制文件。
+
+* `staticlib` (`默认值`) 构建JIT成静态库
+* `lvjitobj` 构建JIT成目标文件
+
+在构建JIT的时候，如果VM作为C代码来编译就必须定义宏 `LUA_C_LINKAGE` 。这可以保证当把JT和VM链接在一起的时候，不会出现命名粉碎规则(name-mangling)问题。当调用 `make`的时候带上参数 `CXX_FLAGS_EXTRA='-DLUA_C_LINKAGE'` 就定义了 `LUA_C_LINKAGE` 宏（参考根目录下的Makefile）。
```
Comment (xliang6) [inline doc/zh/building.md]: 这可以保证当把JT和VM链接在一起的时候，不会出现命名混乱问题。

File: doc/zh/building.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+| 变量                   | 默认值                         | 描述   |
+|:--------------------------:|:-------------------------------------:|:--------------|
+| `JITBUILDER_DIR`           | `$(OMR_DIR)/jitbuilder`               | JitBuilder代码目录 |
+| `JITBUILDER_ARCH_DIR`      | `$(JITBUILDER_DIR)/$(ARCH)`           | JitBuilder特定处理器代码目录 |
+| `JITBUILDER_SUBARCH_DIR`   | `$(JITBUILDER_ARCH_DIR)/$(SUBARCH)`   | JitBuilder特定处理器子架构代码目录 |
+| `JITBUILDER_LIB_DIR`       | `$(JITBUILDER_DIR)/release`           | (见上表)   |
+| `JITBUILDER_HEADERS`       | `$(JITBUILDER_LIB_DIR)/include`       | JitBuilder 头文件目录 |
+| `OMR_COMPILER_DIR`         | `$(OMR_DIR)/compiler`                 | Eclipse OMR 编译器代码目录 |
+| `OMR_COMPILER_ARCH_DIR`    | `$(OMR_COMPILER_DIR)/$(ARCH)`         | Eclipse OMR 编译器CPU特定架构代码 |
```
Comment (xliang6) [inline doc/zh/building.md]: Eclipse OMR 编译器特定架构代码目录

File: doc/zh/building.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+| `OMR_COMPILER_SUBARCH_DIR` | `$(OMR_COMPILER_ARCH_DIR)/$(SUBARCH)` | Eclipse OMR 编译器CPU特定子架构代码 |
```
Comment (xliang6) [inline doc/zh/building.md]: Eclipse OMR 编译器特定子架构代码目录

**Review by Leonardo2718 — CHANGES_REQUESTED** [review]
@leejingbo, please address @xliang6's comments and then I will merge.

File: README_zh.md (hunk 1 above)
Comment (Leonardo2718) [reply README_zh.md]: I'll create a separate issue for this (thanks for reminding me 😛).

**Comment by xliang6** [comment]
Looks good to me. One small suggestion: We may want to squash all these commits into one before merging?

**Comment by Leonardo2718** [comment]
I don't mind having one commit per "logical unit of translation". So I'm okay with one commit per file. But I don't like having multiple commits for one file. So @leejingbo, would you mind squashing those together?

**Comment by leejingbo** [comment]
@Leonardo2718 I rebased my repo and squashed the commits. 

@xliang6 Thank you for your suggestion !

**Review by Leonardo2718 — APPROVED** [review]

**Pull request merged**